client/components/domains/contact-details-form-fields/custom-form-fieldsets/eu-address-fieldset.jsx

```diff
@@ -17,13 +17,13 @@ const EuAddressFieldset = props => {
 		<div className="custom-form-fieldsets__address-fields eu-address-fieldset">
 			<Input
 				label={ translate( 'Postal Code' ) }
-				errorMessage={ contactDetailsErrors?.postalCode }
 				{ ...getFieldProps( 'postal-code' ) }
+				errorMessage={ contactDetailsErrors?.postalCode }
```

Comment:
Does `getFieldProps()` return `errorMessage`? If so, will this override it even if it's `undefined`? Or does React allow that to work?